pkg/apis/kf/v1alpha1/app_types.go

```diff
@@ -117,6 +124,22 @@ type AppSpecInstances struct {
 	Max *int `json:"max,omitempty"`
 }
 
+// AppSpecServiceBinding is a binding to an external service.
+type AppSpecServiceBinding struct {
+
+	// InstanceRef is the service the app will bind to.
+	InstanceRef servicecatalogv1beta1.LocalObjectReference `json:"instanceRef"`
+
+	// Parameters is an arbitrary JSON to be injected into VCAP_SERVICES.
+	// +optional
+	Parameters []byte `json:"parameters,omitempty"`
```

Comment:
I think this should be validated in the validtor to be valid JSON or blank. Go has a function that can do this: https://golang.org/pkg/encoding/json/#Valid